.pre-commit-config.yaml

```diff
@@ -54,3 +54,8 @@ repos:
         pass_filenames: false
         language: python
         additional_dependencies: [PyYAML]
+
+-   repo: https://github.com/MarcoGorelli/cython-lint
```

Comment:
Let's move this section to just after the flake8 (after `https://github.com/PyCQA/flake8` line)